sbncode/Metadata/FileCatalogMetadataSBN.h

```diff
@@ -66,6 +74,8 @@ namespace util {
     std::string fProjectSoftware;
     std::string fProductionName; //Production parameter, do not use if not running a production
     std::string fProductionType; //Production parameter, do not use if not running a production
+    int fMerge;
+    std::vector<std::string> fParameters;
```

Comment:
Can we use `vector<pair<string, string>>` or `map<string, string>` please?

Reply:
The reason I had it as vector<string> is because that is a supported type by fcl.  I'm OK with switching to a pairwise type.